seccomp_internal.go

```diff
@@ -122,6 +137,26 @@ unsigned int get_micro_version()
 }
 #endif
 
+// The libseccomp API level functions were added in v2.3.3
+#if (SCMP_VER_MAJOR < 2) || \
```

Comment:
Should we swap this macro to require 2.4.0 minimum?

Reply:
Nice catch!

Reply:
There were two macros to swap and several comments that needed to be adjusted from `v2.3.3` to `v2.4.0`.